operator/pkg/apis/istio/v1alpha1/values_types.proto

```diff
@@ -88,6 +88,11 @@ message CNIConfig {
   google.protobuf.Struct seccompProfile = 19;
 
   CNIAmbientConfig ambient = 21;
+
+  string provider = 22;
```

Comment:
It looks like this is extra change which is not included in the original PR https://github.com/istio/istio/pull/44839

Reply:
Can you verify if anything should be cherry-picked before this PR?

Reply:
I see, there's another PR that should be cherry-picked before yours. I'm going to put on a hold before https://github.com/istio/istio/issues/44642 is resolved.

Reply:
@hanxiaop Thanks for catching this. I'll rebase this PR when #44642 is resolved.

Reply:
@hanxiaop Since it looks like #44642 may not be included in 1.18, I just reserved the field number in the proto